dbt_platform_helper/constants.py

```diff
@@ -4,6 +4,8 @@
 PLATFORM_HELPER_VERSION_FILE = ".platform-helper-version"
 # Todo: Move to ???
 DEFAULT_TERRAFORM_PLATFORM_MODULES_VERSION = "5"
+SUPPORTED_TERRAFORM_VERSION = "~> 1.8"
```

Comment:
https://uktrade.atlassian.net/wiki/spaces/DBTP/pages/4340056065/ADR009+Draft+-+Support+2+minor+releases+of+Terraform + https://github.com/hashicorp/terraform/releases
```suggestion
SUPPORTED_TERRAFORM_VERSION = "~> 1.9"
```